package.json

```diff
@@ -6,7 +6,8 @@
     "node-sass": "^4.11.0",
     "normalize.css": "^8.0.1",
     "react": "^16.7.0",
-    "react-dom": "^16.7.0"
+    "react-dom": "^16.7.0",
+    "react-modal": "^3.8.1"
```

Comment:
I think you should create your own modal 😉

Reply:
okay, algo a ponderar. Durante esta semana pego e substituiu o react-modal or uma minha !!

> I think you should create your own modal 😉